### ISSUE TITLE
Select which camera is default on a phone

### ISSUE THREAD

**Issue by geekgirl4infinity8** [nstrayer/shinysense#53]
Is it possible to make the rear camera the default when it detects a mobile phone is being used? It would be nice to control that.

**Comment by moldach** [comment]
I agree, that an option to switch between front/rear camera would be a good feature to include

**Comment by doncqueurs** [comment]
This would be a nice feature to have. Given the updates on this GitHub, I assume that the project is not actively maintained?
